### ISSUE TITLE
Representation for native code

### ISSUE THREAD

**Issue by npmccallum** [deislabs/bindle#270]
I know that bindle was designed with WASM in mind. But I thought I'd just toss out something to think about. Bindle currently has no way to represent parcel's that are equivalent binaries for different CPU ISAs. For example, what if you build the same binary for x86_64 and aarch64? How should the client know which to download?

**Comment by radu-matei** [comment]
[The feature selection section in the label spec](https://github.com/deislabs/bindle/blob/main/docs/label-spec.md#the-feature-section) should provide a starting point for this.
I think that if we specify some standard label / feature keys, this could be solved such as when requesting the bindle and a desired value for the label (in this case the architecture), the client would get the proper artifact.

**Comment by npmccallum** [comment]
@radu-matei Indeed. I had a similar thought. However, I'm worried that this will become overloaded with other concerns like build versioning (#266) and build variants (#269). All three of these concerns are vying for the same space.

**Comment by thomastaylor312** [comment]
Yep, see [here](https://wasmcloud.dev/reference/bindle/provider-bindles/#a-full-example) for a full example
